src/components/Login/Login.tsx

```diff
@@ -45,7 +45,8 @@ const Login = () => {
   const [loginMutation, { loading }] = useMutation<AuthData>(LOGIN_MUTATION, {
     onCompleted: ({ login }) => {
       auth.setToken(login.token);
-      history.push(ROUTES.BUDDY_SELECT_NEWBIE);
+      auth.setUser({ id: login.userId, role: login.role });
```

Comment:
`auth.setToken` and `auth. setUser ` should be changed to use `AuthContext`. We have create separate folder `/src/context` with Auth part